server-side-rendering/src/routes/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	export let data;
+
+	const { user } = data;
+
+	function getInitials(name: string) {
+		const [first, last] = name.split(' ');
+		if (last) return `${first[0]}${last[0]}`;
+		return `${first[0]}`;
+	}
+</script>
+
+<div class="u-max-width-500 u-width-full-line">
+	<h1 class="heading-level-2 u-margin-block-start-auto">Your account 🎉</h1>
```

Comment:
No emoji, confirm with Sara if unsure. I know our tone is against emojis after rebrand